pennylane/devices/default_qubit.py

```diff
@@ -116,6 +116,24 @@ class DefaultQubit(Device):
 
     where the last two are specific to the MKL and OpenBLAS libraries specifically.
 
+    .. warning::
+
+        Multiprocessing may fail depending on your platform and environment (Python shell,
+        script with a protected entry point, Jupyter notebook, etc.) This may be solved
+        changing the so-called start method. The supported start methods are the following:
+
+        * Windows (win32): spawn (default).
+        * macOS (darwin): spawn (default), fork, forkserver.
+        * Linux (unix): spawn, fork (default), forkserver.
+
+        which can be changed with ``multiprocessing.set_start_method()``. For example,
+        if multiprocessing fails on macOS in your Jupyter notebook environment, try
+        restarting the session, importing multiprocessing and running
+        ``multiprocessing.set_start_method("fork")`` before doing anything.
```

Comment:
It might be better to add this example in a code block.

Reply:
Like that?
![Screenshot from 2023-09-21 15-58-03](https://github.com/PennyLaneAI/pennylane/assets/8711156/1d6df2fd-f4e8-4ead-92d8-57e5a11d8dbc)